src/components/Autocomplete/Autocomplete.tsx

```diff
@@ -20,20 +20,23 @@ import { PlaceholderProps } from 'react-select/lib/components/Placeholder';
 import { OptionProps } from 'react-select/lib/components/Option';
 import { NoticeProps, MenuListComponentProps, MenuProps } from 'react-select/lib/components/Menu';
 import { MultiValueProps } from 'react-select/lib/components/MultiValue';
-import { getOptionValue, getOptionLabel } from 'react-select/lib/builtins';
 import { ControlProps } from 'react-select/lib/components/Control';
 import { IndicatorProps } from 'react-select/lib/components/indicators';
-import { StylesConfig } from 'react-select/lib/styles';
-import { SelectComponents } from 'react-select/lib/components';
 
 import { ChevronDown } from '../Icons';
 import './Autocomplete.scss';
 
-const styles = ({ spacing, palette, transitions }: Theme) =>
-  createStyles({
+const styles = ({ spacing, palette, transitions }: Theme) => {
+  const borderColor =
```

Comment:
This is how they do it in the Material-ui repo for determining what color the underline of an input should be :/. Not sure why this isn't part of the theme.